elements/bulbs-pinned-element/bulbs-pinned-element-examples.js

```diff
@@ -4,10 +4,17 @@ let examples = {
     'Basic Example': {
       render: () => {
         return `
-          <bulbs-pinned-element
-            attribute="value"
-          >
-          </bulbs-pinned-element>
+          <div
```

Comment:
This example doesn't work at this point since pinned element tracks on window scroll instead of in relation to a relative scrolling container. This will be logged as a known issue.